tailcfg/tailcfg.go

```diff
@@ -680,6 +680,11 @@ type Location struct {
 	// IATA, ICAO or ISO 3166-2 codes are recommended ("YSE")
 	CityCode string `json:",omitempty"`
 
+	// Latitude, Longitude are optional geographical coordinates in degrees, of the node.
```

Comment:
optional: I think this is less awkward

```suggestion
	// Latitude, Longitude are optional geographical coordinates of the node, in degrees.
```